packages/lab/stories/progress/progress.circular.stories.tsx

```diff
@@ -89,86 +88,49 @@ export const All: ComponentStory<typeof CircularProgress> = () => (
 );
 
 export const Default: ComponentStory<typeof CircularProgress> = () => (
-  <div style={{ display: "flex" }}>
-    <div>
-      <h3>size=small</h3>
-      <CircularProgress aria-label="Download" size="small" value={100} />
-    </div>
-    <div style={{ marginLeft: 100 }}>
-      <h3>size=medium</h3>
-      <CircularProgress aria-label="Download" size="medium" value={38} />
-    </div>
-    <div style={{ marginLeft: 100 }}>
-      <h3>size=large</h3>
-      <CircularProgress aria-label="Download" size="large" value={38} />
-    </div>
-  </div>
+  <CircularProgress aria-label="Download" value={38} />
 );
 
 export const Disabled: ComponentStory<typeof CircularProgress> = () => (
-  <div style={{ display: "flex" }}>
-    <div>
-      <h3>size=small</h3>
-      <CircularProgress
-        aria-label="Download"
-        disabled
-        size="small"
-        value={38}
-      />
-    </div>
-    <div style={{ marginLeft: 100 }}>
-      <h3>size=medium</h3>
-      <CircularProgress
-        aria-label="Download"
-        disabled
-        size="medium"
-        value={38}
-      />
-    </div>
-    <div style={{ marginLeft: 100 }}>
-      <h3>size=large</h3>
-      <CircularProgress
-        aria-label="Download"
-        disabled
-        size="large"
-        value={38}
-      />
-    </div>
-  </div>
+  <CircularProgress aria-label="Download" disabled value={38} />
 );
 
 export const Indeterminate: ComponentStory<typeof CircularProgress> = () => (
   <div style={{ display: "flex" }}>
     <div>
-      <h3>size=small</h3>
-      <CircularProgress
-        aria-label="Download"
-        showInfo={false}
-        size="small"
-        variant="indeterminate"
-      />
+      <h3>HD</h3>
+      <SaltProvider density="high">
+        <CircularProgress aria-label="Download" variant="indeterminate" />
+      </SaltProvider>
+    </div>
+    <div style={{ marginLeft: 100 }}>
+      <h3>MD</h3>
+      <SaltProvider density="medium">
+        <CircularProgress aria-label="Download" variant="indeterminate" />
+      </SaltProvider>
     </div>
     <div style={{ marginLeft: 100 }}>
-      <h3>size=medium</h3>
-      <CircularProgress
-        aria-label="Download"
-        showInfo={false}
-        size="medium"
-        variant="indeterminate"
-      />
+      <h3>LD</h3>
+      <SaltProvider density="low">
+        <CircularProgress aria-label="Download" variant="indeterminate" />
+      </SaltProvider>
     </div>
     <div style={{ marginLeft: 100 }}>
-      <h3>size=large</h3>
-      <CircularProgress
-        aria-label="Download"
-        showInfo={false}
-        size="large"
-        variant="indeterminate"
-      />
+      <h3>TD</h3>
+      <SaltProvider density="touch">
+        <CircularProgress aria-label="Download" variant="indeterminate" />
+      </SaltProvider>
     </div>
   </div>
```

Comment:
I would change this to be simply <CircularProgress aria-label="Download" variant="indeterminate" />
the density can be changed in storybook, since this story is quite old now, i think thats why all of the densities are shown at once here. but we tend to now just show the one example and change through toggles :)